docs/recipes/ec2-eks-metrics-go-adot-ampamg.md

```diff
@@ -1,12 +1,314 @@
+
+Use the following steps to edit the downloaded file for your environment:
+
+1\. Replace **<REGION\>** with your current Region. 
+
+2\. Replace **<YOUR_ENDPOINT\>**  with your AMP workspace endpoint URL.
+
+Get your AMP endpoint url by executing the following query:
+```
+$aws amp describe-workspace \ 
+    --workspace-id `aws amp list-workspaces --alias prometheus-sample-app --query 'workspaces[0].workspaceId' --output text` \
+    --query 'workspace.prometheusEndpoint'
+```
+
+3\. Finally replace your **<YOUR_ACCOUNT_ID\>**  with your current account ID.
+
+The following command will return the account ID for the current session:
+```
+aws sts get-caller-identity --query Account --output text
+```
+<br/>
+
+#### Deploy the template to your cluster
+
+```
+$ kubectl apply -f prometheus-fargate.yaml
+```
+
+In case you used the deamonset template use the following command
+```
+$ kubectl apply -f prometheus-daemonset.yaml
+```
+
+You can verify that the ADOT Collector has started with this command:
+
+```
+$ kubectl get pods -n adot-col
+```
+
+For more information check out the [AWS Distro for OpenTelemetry (ADOT) Collector Setup](https://aws-otel.github.io/docs/getting-started/prometheus-remote-write-exporter/eks#aws-distro-for-opentelemetry-adot-collector-setup)
+
+### Setup AMG
+
+[Amazon Managed Service for Grafana – Getting Started](https://aws.amazon.com/blogs/mt/amazon-managed-grafana-getting-started/)
+
+Setup a new AMG workspace using the Getting Started guide above.
+
+Make sure to add "Amazon Managed Service for Prometheus" as a datasource during creation.
+
+![Service managed permission settings](https://d2908q01vomqb2.cloudfront.net/972a67c48192728a34979d9a35164c1295401b71/2020/12/09/image008-1024x870.jpg)
+
+
 ## Application
 
+The following sample application will be used in this guide:
+
+[Sample App Github](https://github.com/aws-observability/aws-otel-community/tree/master/sample-apps/prometheus)
+
+
+### Build
+Clone the following Git repository
+```
+$ git clone git@github.com:aws-observability/aws-otel-community.git
+```
+
+Build the container
+```
+$ cd ./aws-otel-community/sample-apps/prometheus
+$ docker build . -t prometheus-sample-app:latest
+```
+
+### Push
+Authenticate to your default registry:
+```
+aws ecr get-login-password --region region | docker login --username AWS --password-stdin aws_account_id.dkr.ecr.region.amazonaws.com
+```
+
+Push container to the ECR repository
+```
+docker tag prometheus-sample-app:latest <aws_account_id>.dkr.ecr.eu-west-1.amazonaws.com/prometheus-sample-app:latest
+docker push <aws_account_id>.dkr.ecr.eu-west-1.amazonaws.com/prometheus-sample-app:latest
+```
+
+### Deploy
+Edit prometheus-sample-app.yaml to contain your ECR image path.
+
+Deploy the sample app to your cluster:
+```
+$ kubectl apply -f prometheus-sample-app.yaml
```

Comment:
This command will have the same challenge as prometheus-fargate.yaml if fargate profile chosen, I think it is a better idea to work this out.

Reply:
Resolved this one by adding it to the cluster-config.yml in the beginning just like the adot-col.